plugins/woocommerce-blocks/tests/e2e/tests/product-filters/price-filter-editor.block_theme.spec.ts

```diff
@@ -1,66 +0,0 @@
-/**
- * External dependencies
- */
-import { test as base, expect } from '@woocommerce/e2e-utils';
-
-/**
- * Internal dependencies
- */
-import { ProductFiltersPage } from './product-filters.page';
-
-const blockData = {
-	name: 'woocommerce/product-filter-price',
-	selectors: {
-		frontend: {},
-		editor: {
-			settings: {},
-		},
-	},
-	slug: 'archive-product',
-};
-
-const test = base.extend< { pageObject: ProductFiltersPage } >( {
-	pageObject: async ( { page, editor, frontendUtils }, use ) => {
-		const pageObject = new ProductFiltersPage( {
-			page,
-			editor,
-			frontendUtils,
-		} );
-		await use( pageObject );
-	},
-} );
-
-test.describe( `${ blockData.name }`, () => {
-	test.beforeEach( async ( { admin, requestUtils } ) => {
-		await requestUtils.setFeatureFlag( 'experimental-blocks', true );
-		await admin.visitSiteEditor( {
-			postId: `woocommerce/woocommerce//${ blockData.slug }`,
-			postType: 'wp_template',
-			canvas: 'edit',
-		} );
-	} );
-
-	test( 'should be able to enable or disable the "Clear" button for the filter', async ( {
-		editor,
-		pageObject,
-	} ) => {
-		await pageObject.addProductFiltersBlock( { cleanContent: true } );
-
-		const block = editor.canvas.getByLabel( 'Block: Price (Experimental)' );
-		await expect( block ).toBeVisible();
```

Comment:
The test says it's about Clear button but implicitly it was also testing if the filter block can be added in editor. I suggest bringing it back at in least in form of basic test, something like:

```
test( 'block can be added in Editor', async ( {
		editor,
		pageObject,
	} ) => {
		await pageObject.addProductFiltersBlock( { cleanContent: true } );

		const block = editor.canvas.getByLabel( 'Block: Price (Experimental)' );
		await expect( block ).toBeVisible();
        } );
} );
```